tdrs-backend/tdpservice/reports/models.py

```diff
@@ -61,31 +87,45 @@ class Meta:
 
         constraints = [
             models.UniqueConstraint(
-                fields=("section", "version", "quarter", "year", "stt"),
-                name="constraint_name",
+                fields=("version", "quarter", "year", "stt"),
+                name="unique_reports_reportfile_fields",
             )
         ]
 
+    # Override FileRecord fields
+    extension = models.CharField(max_length=8, default="zip")
+
+    # Model Fields
     created_at = models.DateTimeField(auto_now_add=True)
     quarter = models.CharField(
         max_length=16, blank=False, null=False, choices=Quarter.choices
     )
     year = models.IntegerField()
-    section = models.CharField(
-        max_length=32, blank=False, null=False, choices=Section.choices
-    )
 
     version = models.IntegerField()
 
     user = models.ForeignKey(
-        User, on_delete=models.CASCADE, related_name="user", blank=False, null=False
+        User,
+        on_delete=models.CASCADE,
```

Comment:
I think this should be set to null to. I don't think we ever want to delete these. Same for the STT FK
```suggestion
        on_delete=models.SET_NULL,
```

Reply:
Done.